spec/factories.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user do
+    email { 'joe@gmail.com' }
```

Comment:
Since `User#email` needs to be unique, this will crash on a second user created in a test. You must change the email to `sequence`